scikit/notebooks/Scikit Learn on CMLE.ipynb

```diff
@@ -0,0 +1,381 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Predicting Income with the Census Income Dataset Using Scikit Learn on ML Engine\n",
+    "To use Scikit Learn on ML Engine for predictions, you will need to have a pre-trained model that you can upload to ML Engine. This datalab use the [Census Income Data Set](https://archive.ics.uci.edu/ml/datasets/Census+Income) to create a simple model, train the model, upload the model to ML Engine, and lastly use the model to make predictions. "
```

Comment:
The UC Irvine dataset has some header text. We have a sanitized copy of it on GCS. We should ask users to download from GCS (see other census for location)

Reply:
The adult.data.csv vs adult.test.csv data on the GCS was heavily skewed in that there was nobody that made >50K in the test data set.

Reply:
Really? It should be same data except some cleaning.

Reply:
I think the issue here is that, in the raw data there are leading spaces in each of the fields, so `">50K"` is actually `" >50K"` in the raw data. The CSV files offered on the cloudml bucket are preprocessed to strip these spaces, I think.

I think it's best to stick with the raw data from the UCI repository.

Reply:
@puneith : To provide a little context - the header text is actually a positive when working with `pandas`.

Reply:
Not sure what's positive about the header for pandas. Also train doesn't have it and test has it. Additionally the trailing spaces. Moreover this was a feedback from a hackathon which users were confused about so we had to fix it.

Reply:
Positive is that you don't have to spend much time figuring out what your dataframe contains as `pandas` displays the headers as a result of `read_csv`